src/qililab/constants.py

```diff
@@ -108,7 +106,6 @@ class EXPERIMENT:
     NUM_SCHEDULES = "num_schedules"
     LOOPS = "loops"
     OPTIONS = "options"
-    DEVICE_ID = "device_id"
     REMOTE_DEVICE_MANUAL_OVERRIDE = "remote_device_manual_override"
     CONNECTION = "connection"
     CIRCUITS = "circuits"
```

Comment:
And also, 2 lines below with the `REMOTE_SAVE`, one 🙃

Reply:
Done!